packages/interface/src/features/home/components/FaqList.tsx

```diff
@@ -59,12 +57,11 @@ export const FAQList = (): JSX.Element => (
       description={
         <div className="flex flex-col gap-4">
           <p>
-            Join our{" "}
-
-            <Link className="font-bold underline" href="https://discord.gg/Bj9PWNVu" rel="noreferrer" target="_blank">
+            Join our
+            <a className="font-bold underline" href="https://discord.gg/Bj9PWNVu" rel="noreferrer" target="_blank">
```

Comment:
why change the `Link` to `<a>`?

Reply:
Some issues with the linter and the prettier not agreeing in some changes which made some tests fail.